### DISCUSSION TITLE
Standalone install possible?

### DISCUSSION THREAD

**Discussion by myxr6turbo** [Lissy93/dashy#1188] [Help and Support]
Hi guys, I would like to install Dashy in our ESXi environment on a Ubuntu server. I have installed it but I get the splash screen and then it says possible error. I'm unable to locate any log files. Can anyone help me? Do I need to enable logging? Below are the commands I ran to install it.

sudo apt-get install nodejs npm
git clone https://github.com/Lissy93/dashy.git
npm install
npm start

Thanks in advance,

**Comment by Glitch3dPenguin** [comment]
It's not first party supported but I did make a script to install this directly in a Proxmox LXC container. Standalone - no Docker required. 
 
https://github.com/Glitch3dPenguin/DashyDashboardForLXC

All in a simple "oneliner".
